enterprise/tiny-comments.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: default
+title: Tiny Comments
+description: Tiny Comments provides the ability to add comments to the content and collaborate with other users for content editing.
+keywords: enterprise pricing video youtube vimeo mp3 mp4 mov movie clip film comment commenting mediaembed media
+---
+
+The [Tiny Comment]({{ site.baseurl }}/plugins/tiny-comment/) plugin provides the ability to add comments to the content and collaborate with other users for content editing within the TinyMCE editor.
+The primary value that TinyComments offer is a user interface to create, reply, and delete comments. The TinyComments plugin is built upon the the new [Annotations API]({{ site.baseurl }}/plugins/annotations/) and uses annotations to create comment threads.
```

Comment:
- "The primary value"? What about just "Tiny Comments offers a user interface.."
- If you keep it the way it is, it should be "The primary value that Tiny Comments offer**s**"